examples/rmrk/resource/wasm/tests/resources.rs

```diff
@@ -219,6 +304,10 @@ impl<'a> Fixture<'a> {
         &self.program_space
     }
 
+    fn resource_client(&self) -> crate::resource_client::RmrkResource<GTestRemoting, GTestArgs> {
```

Comment:
you can even add a type alias for RmrkResourceClient in order to avoid writting generic params all the time
```suggestion
    fn resource_client(&self) -> RmrkResourceClient<GTestRemoting, GTestArgs> {
```

Reply:
added type alias RmrkResourceClient